test/stdlib/TestCharacterSet.swift

```diff
@@ -192,6 +192,15 @@ class TestCharacterSet : TestCharacterSetSuper {
         expectEqual(expected, union)
     }
 
+    func test_subtractAndFormSymmetricDifference() {
```

Comment:
Thanks @xwu . I just realized that it is not really clear that the test is doing the right thing because of mutation. `set1.subtract(set2)` modifies `set`. Can you please split the test into several, one test per case. In order to make absolutely clear what the result of operation is, I would also assert on the set equality, instead of just checking whether a particular element is in the set.